docs/AboutUs.md

```diff
@@ -3,7 +3,7 @@
 | Display                                             |       Name        |             Github Profile              |              Portfolio               |
 |-----------------------------------------------------|:-----------------:|:---------------------------------------:|:------------------------------------:|
 | ![](https://via.placeholder.com/100.png?text=Photo) |     Ming Chun     | [Github](https://github.com/mcmc101001) | [Portfolio](docs/team/mcmc101001.md) |
-| ![](https://via.placeholder.com/100.png?text=Photo) |    Shan YuXuan    |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
+| ![](https://via.placeholder.com/100.png?text=Photo) |    Shan YuXuan    | [Github](https://github.com/xuan127)    |  [Portfolio](team/xuan127.md)   |
```

Comment:
Maybe we should also all add a profile picture to make this page look more professional